docs/install/cluster.md

```diff
@@ -0,0 +1,258 @@
+# Multinode Deployment with Docker Compose
```

Comment:
We should mark experimental?

Reply:
I'm mentioning that this feature will improve in future releases (e.g. follower will be able to ingest data), so maybe it's a good idea.

Reply:
Done:
https://github.com/graphistry/graphistry-cli/pull/52/commits/ed29797213c5ea0e71ac6937fb4f90d3a66507b2#diff-85154aa760ae77767d3ae57d36e9bc095a03dc613694339d7a919b991a345e51R3